services/powchain/src/main/java/tech/pegasys/teku/services/executionengine/client/schema/PayloadAttributesV1.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.services.executionengine.client.schema;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.services.executionengine.client.serialization.Bytes20Deserializer;
+import tech.pegasys.teku.services.executionengine.client.serialization.Bytes20Serializer;
+import tech.pegasys.teku.services.executionengine.client.serialization.Bytes32Deserializer;
+import tech.pegasys.teku.services.executionengine.client.serialization.BytesSerializer;
+import tech.pegasys.teku.services.executionengine.client.serialization.UInt64AsHexDeserializer;
+import tech.pegasys.teku.services.executionengine.client.serialization.UInt64AsHexSerializer;
+import tech.pegasys.teku.ssz.type.Bytes20;
+
+public class PayloadAttributesV1 {
```

Comment:
We should add tests to confirm the serialisation and deserialisation of these new types (and probably some simple tests for each of the `*Serializer` and `*Deserializer` classes as well.